updated the approved forms for Consent and HIPAA

ramavavi opened the pull request:
Git Hub Issue: https://github.com/episphere/connectApp/issues/491

Updated the consent and HIPAA forms for all the sites. ( 4 sites remaining for the approved forms).

anthonypetersen reviewed (commented):
The HP HIPAA PDF file has a dash before the version number instead of an underscore and the HFHS Consent HTML file is still using V1.0

ramavavi commented:
I was talking about the HP HIPAA pdf, but I renamed it. HFHS Consent HTML version 2 is also added, but in the files changed its also showing version 1 as I removed the space between the lines and that counted as a change in the code. 

[HFHS Consent HTML Version 0.02](https://github.com/episphere/connectApp/blob/75ebd8af778f7c0c59a95940204dfbb4b38d88fd/forms/consent/HFHS_Consent_V0.02.html)


[HFHS Consent HTML Version 1.0](https://github.com/episphere/connectApp/blob/75ebd8af778f7c0c59a95940204dfbb4b38d88fd/forms/consent/HFHS_Consent_V1.0.html)
